### PR TITLE
feat: added homepage with css

### PR DESCRIPTION
**Pull request by paarthbajaj**
homepage

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+  --dark-blue: #0751a6;
+  --blue-shade1: #024ca1;
+}
+.desktop-header-container {
+  /* color: #fff; */
+  top: 0;
+  position: fixed;
+  right: 0;
+  left: 0;
+  z-index: 999;
```
Comment (harshitadaswani) [inline homepage.css]: use lower values

File: homepage.css [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
+  height: 4.4rem;
+  background-color: var(--white);
+  /* margin: auto; */
```
Comment (harshitadaswani) [inline homepage.css]: remove comments

File: homepage.css [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  font-weight: 500;
+  color: var(--black);
+}
+.flex {
+  display: flex;
+}
+.input-search-bar {
+  font-size: 0.875rem;
+  /* height: 20px;
+  line-height: 24px; */
```
Comment (harshitadaswani) [inline homepage.css]: remove comments

File: homepage.css [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+  text-transform: uppercase;
+  letter-spacing: 0.5px;
+}
+.navbar {
+  margin-left: 3%;
+  flex: 0 1 auto;
+}
+.nav-items {
+  text-align: center;
+  padding: 0 17px 28px;
```
Comment (harshitadaswani) [inline homepage.css]: use rems

File: homepage.css [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+}
+.action-link {
+  text-align: center;
+}
+.main-content {
+  margin-top: 4.3rem;
+}
+.hero-section {
+  /* width: 100%;
+  height: 100vh; */
```
Comment (harshitadaswani) [inline homepage.css]: remove comments

File: homepage.css [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+  object-fit: cover;
+  width: 100%;
+  height: 90vh;
+}
+
+.features-card {
+  justify-content: center;
+  padding: 1.5rem 0 3rem 0;
+  font-size: 1.3rem;
+  /* align-items: center; */
```
Comment (harshitadaswani) [inline homepage.css]: remove comments

File: homepage.css [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+  background-color: var(--gray-shade1);
+  position: relative;
+}
+.single-category h4 {
+  position: absolute;
+  left: 4px;
+  bottom: 4px;
+  padding: 0.5rem 1rem;
+  border-radius: 5px;
+  background-color: #c7c7c7e5;
```
Comment (harshitadaswani) [inline homepage.css]: use color variable

File: homepage.css [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+  max-height: 350px;
+  min-width: 350px;
+  max-width: 350px;
+}
+.top-selling {
+  text-align: center;
+  margin: 5rem 0;
+}
+.ts-h5 {
+  font-size: 14px;
```
Comment (harshitadaswani) [inline homepage.css]: suggested to use `rems` to make it responsive

File: index.html [hunk 9]
```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./homepage.css" />
```
Comment (harshitadaswani) [inline index.html:7]: suggested to use component library stylesheet too as it is required for assignment submissions

**Review by harshitadaswani — CHANGES_REQUESTED** [review]
Refactor the suggested changes

File: homepage.css (hunk 8 above)
Comment (paarthbajaj) [reply homepage.css]: Thank you for reviewing the code.

File: index.html (hunk 9 above)
Comment (paarthbajaj) [reply index.html:7]: Changes done

**Review by shaksham08 — APPROVED** [review]
Looks good to me

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge. But try to use less fixed values and more of properties of flex and grid to align stuff.

**Review by harshitadaswani — APPROVED** [review]
Yes, you can merge this.

**Pull request merged**